tutorials/btp-app-events-app-setup-s4hc/btp-app-events-app-setup-s4hc.md

```diff
@@ -0,0 +1,378 @@
+ - On SAP BTP side:
+    - You have an [enterprise](https://help.sap.com/viewer/65de2977205c403bbc107264b8eccf4b/Cloud/en-US/171511cc425c4e079d0684936486eee6.html) global account in SAP BTP.
+    - You must be an administrator of the SAP BTP global account where you want to register your SAP S/4HANA Cloud system.
+    - You need to [Prepare for SAP BTP Development](btp-app-prepare-btp) if you start with a result from an example branch.
+    - Your SAP BTP subaccount has quota for the services `SAP Launchpad service` and `SAP HTML5 Applications Repository service` as described in [Prepare for SAP BTP Development](btp-app-prepare-btp).
+    - You have to [Use an existing SAP HANA Cloud service instance](https://developers.sap.com/tutorials/btp-app-hana-cloud-setup.html#42a0e8d7-8593-48f1-9a0e-67ef7ee4df18) or [Set up a new SAP HANA Cloud service instance](https://developers.sap.com/tutorials/btp-app-hana-cloud-setup.html#3b20e31c-e9eb-44f7-98ed-ceabfd9e586e) for the deployment. After the deployment, you need to perform step 14–17, starting with step [Subscribe to SAP Launchpad Service](https://developers.sap.com/de/tutorials/btp-app-launchpad-service.html#57352c79-1a09-4054-a77d-626fac957404) from the tutorial [Add the SAP Launchpad service](btp-app-launchpad-service).
+ - On SAP S/4HANA Cloud side:
+    - You have a dedicated SAP S/4HANA Cloud tenant.
+    - You must be an administrator of the SAP S/4HANA Cloud system.
+    - You need to connect this system to your SAP BTP global account, if you'd like to build extension applications for your SAP S/4HANA Cloud system.
+
+
+## Details
+### You will learn
+ - How to configure your CAP application for eventing
+ - How to locally test your changes
+ - How to deploy your CAP application
+
+
+To start with this tutorial use the result in the [`ext-service-s4hc-use`](https://github.com/SAP-samples/cloud-cap-risk-management/tree/ext-service-s4hc-use) branch.
+
+---
+
+[ACCORDION-BEGIN [Step 1: ](Configure your CAP application for eventing)]
+CAP provides native support for emitting and receiving events. Hence, CAP applications can receive events for changes in remote systems. In CAP, the messaging follows a publish subscribe (pub/sub) model. Pub/sub means that sources publish messages and interested consumers can subscribe to receive them. This helps with scalability, decoupling of services, and robustness.
+
+Using the Business Partner service that we added in `Step 1: Get the Business Partner service definition` from the tutorial [Add the Consumption of an External Service to Your CAP Application](btp-app-ext-service-add-consumption), we will configure the CAP application to listen and react to events from your SAP S/4HANA Cloud system. As in the previous tutorials, let's again use the example of business partners that are suppliers. So, we'll use suppliers when referring to business partners. First we'll add event definitions for the SAP S/4HANA events.
```

Comment:
Please do not use "we" Just use the action verb (e.g., "that was added" and "you will configure" and "again use the example" and "So, use suppliers"